boxsdk/object/file.py

```diff
@@ -763,6 +764,31 @@ def get_thumbnail(self, extension='png', min_width=None, min_height=None, max_wi
         response = self._session.get(url, params=params, expect_json_response=False)
         return response.content
 
+    @api_call
+    def get_thumbnail_representation(self, dimensions, extension='png'):
+        """
+        Retrieve a thumbnail image for the file.
+
+        :param dimensions:
+            The width by height size of this representation in pixels
+        :type dimensions:
+            `unicode`
+        :param extension:
+            The file extension for the thumbnail, e.g. 'png' or 'jpg'
+        :type extension:
+            `unicode`
+        :returns:
+            The file contents of the thumbnail image
+        :rtype:
+            `bytes`
+        """
+        rep_hints = '[{0}?dimensions={1}]'.format(extension, dimensions)
```

Comment:
Are dimensions required, or can you make a request for a representation without specifying dimensions?

Reply:
Added some documentation. Dimensions are required. https://developer.box.com/guides/representations/thumbnail-representation/